wagtail_localize/tests/test_edit_translation.py

```diff
@@ -3212,6 +3212,37 @@ def test_cant_machine_translate_without_snippet_perms(self):
 
         assert_permission_denied(self, response)
 
+    @override_settings(
+        WAGTAILLOCALIZE_MACHINE_TRANSLATOR={
+            "CLASS": "wagtail_localize.machine_translators.deepl.DeepLTranslator",
+            "OPTIONS": {
+                "AUTH_KEY": "00000000-aaaa-bbbb-1111-cccccccccccc:fx",
+                "API_ENDPOINT": "https://api-free.deepl.com/v2/translate",
+            },
+        }
+    )
+    def test_deepl_machine_translate_page(self):
+        response = self.client.post(
```

Comment:
Correct me if I'm wrong, but this will actually make and API request to the DEEPL endpoint. We should avoid this and instead mock the response